podpac/core/node.py

```diff
@@ -95,17 +104,39 @@ class Node(tl.HasTraits):
 
     units = Units(default_value=None, allow_none=True)
     dtype = tl.Any(default_value=float)
+    cache_output = tl.Bool(True)
+    cache_update = tl.Bool(False)
```

Comment:
I need this flag for when I cache a completely nan tile, for when I'm offline and request something (which I can't get from SMAP). So, when I'm back online, I want to update the cache instead of retrieving nans from it.